endpoint/common.php

```diff
@@ -480,9 +480,9 @@ function write_to_kinesis($name, $content, $output_message){
   }
 
   $kinesisClient = new Aws\Kinesis\KinesisClient([
-      'profile' => 'default',
       'version' => '2013-12-02',
-      'region' => 'eu-west-1'
+      'region' => 'eu-west-1',
```

Comment:
By preference I would take this from the configuration file too, though I guess it does not matter too much in this instance.  The configuration file can then be set in the cloudformation via the ${AWS::Region} substitution which is automatically set to the region in which the cfn is being deployed